app/assistify/configuration/server/roles.js

```diff
@@ -218,6 +218,7 @@ const createManagerRole = function() {
 
 	const permissions = [
 		'add-user-to-any-c-room',
+		'assign-roles',
```

Comment:
how does this concern performance?

Reply:
This change comes from an older commit not related to Smarti. It was on the `develop` branch and not merged so far, I did not see a reason to exclude it

Reply:
Because it is semantically not related to this PR?

Reply:
This PR is a release. We did not separately merge these changes to `develop`, this is a PR towards `master`

Reply:
The title and description of this PR confuse me. I thought this was a hotfix release. IMHO hotfixes should only contain code related to a specific thing. Patching or rolling back should have no other effect than to change exactly that one particular thing.

Reply:
Agreed. The title is shitty.

Reply:
Now I'm totally confused. If this PR is a hotfix it should only contain the minimal changes to fix the resync issue and merge it to `master` directly. Otherwise we should create a PR (as the title and the description look like) merge it to `develop` first.